internal/api/handlers/schedulers_handler_test.go

```diff
@@ -55,16 +57,20 @@ import (
 
 func TestListSchedulers(t *testing.T) {
```

Comment:
WDYT about adding a test case where the "GetSchedulersWithFilter" method returns an error?